Sources/SwiftGenKit/Stencil/Files+Context.swift

```diff
@@ -0,0 +1,60 @@
+//
+// SwiftGenKit
+// Copyright © 2020 SwiftGen
+// MIT Licence
+//
+
+import Foundation
+import PathKit
+
+extension Files.Parser {
+  public func stencilContext() -> [String: Any] {
+    let files = self.files
+      .sorted { lhs, rhs in lhs.name < rhs.name }
```

Comment:
May want to sort case insensitively, like:
```suggestion
      .sorted { $0.name.lowercased() < $1.name.lowercased() }
```
Note: based on strings context.